src/components/filmPopups/FilmDeletePopup.tsx

```diff
@@ -1,16 +1,19 @@
 import React from "react";
-import { Bg, BgForm, Form, BtnPopupClose, BtnSubmit } from "./filmPopups.styled";
-import { Container } from "../../styles/general";
-import { actionControlVisibility, useDispatch, useSelector } from "../../context/modalMovieContext";
+import {Bg, BgForm, Form, BtnPopupClose, BtnSubmit} from "./filmPopups.styled";
+import {Container} from "../../styles/general";
+import {actionControlVisibility, useDispatch, useSelector} from "../../context/modalMovieContext";
+import {IType} from "./interfaces";
 
 
-const FilmDeletePopup = ({type}: {type: string}) => {
+const FilmDeletePopup = ({type}: IType) => {
+
+  // having troubles typing props here
   const visible = useSelector(({[type]: visibility}) => visibility),
```

Comment:
Just for your awareness, comma-separated const declarations are neat and nice, but linters, most probably, are going to yell at you, so you shouldn't get used to that for your real-world projects.